### PR TITLE
add option to save the rois

### PR DESCRIPTION
**Pull request by jburel**
Add a new version to the idr0062 notebook
The notebook shows
- how to load the data from an OMERO.server
- how to segment the data using StarDist
- how to convert the StarDist labels into OME rois.
- how to save the rois back to the server

### CODE REVIEW

File: notebooks/idr0062_prediction_save.ipynb [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "silent-webmaster",
+   "metadata": {},
+   "source": [
+    "# Load Image from an OMERO.server, segment using StarDist and save the results back to the server.\n",
+    "\n",
+    "The notebook shows:\n",
+    "\n",
+    " - how to load image from an OMERO server.\n",
```
Comment (pwalczysko) [inline notebooks/idr0062_prediction_save.ipynb]:
```suggestion
    " - How to load an image from an OMERO server.\n",
```

File: notebooks/idr0062_prediction_save.ipynb [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+    " - how to segment using a DeepLearning software, [StarDist](https://github.com/stardist/stardist).\n",
```
Comment (pwalczysko) [inline notebooks/idr0062_prediction_save.ipynb]:
```suggestion
    " - How to segment using a Deep Learning software, [StarDist](https://github.com/stardist/stardist).\n",
```

File: notebooks/idr0062_prediction_save.ipynb [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+    " - how to convert the stardist labels into OME rois.\n",
```
Comment (pwalczysko) [inline notebooks/idr0062_prediction_save.ipynb]:
```suggestion
    " - How to convert the stardist labels into OME ROIs.\n",
```

File: notebooks/idr0062_prediction_save.ipynb [hunk 4]
```diff
@@ -0,0 +1,455 @@
...
+    " - how to save them back to the OMERO.server.\n",
```
Comment (pwalczysko) [inline notebooks/idr0062_prediction_save.ipynb]:
```suggestion
    " - How to save the ROIs back to the OMERO.server.\n",
```

File: notebooks/idr0062_prediction_save.ipynb [hunk 5]
```diff
@@ -0,0 +1,455 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "silent-webmaster",
+   "metadata": {},
+   "source": [
+    "# Load Image from an OMERO.server, segment using StarDist and save the results back to the server.\n",
```
Comment (pwalczysko) [inline notebooks/idr0062_prediction_save.ipynb]:
```suggestion
    "# Load an Image from an OMERO.server, segment it using StarDist and save the results back to the server.\n",
```

File: notebooks/idr0062_prediction_save.ipynb [hunk 6]
```diff
@@ -0,0 +1,455 @@
...
+    "\n",
+    " - how to load image from an OMERO server.\n",
+    " - how to segment using a DeepLearning software, [StarDist](https://github.com/stardist/stardist).\n",
+    " - how to convert the stardist labels into OME rois.\n",
+    " - how to save them back to the OMERO.server.\n",
+    "\n",
+    "The image is referenced in the paper \"NesSys: a novel method for accurate nuclear segmentation in 3D\" published August 2019 in PLOS Biology: https://doi.org/10.1371/journal.pbio.3000388 and can be viewed online in the [Image Data Resource](https://idr.openmicroscopy.org/webclient/?show=image-6001247).\n",
+    "\n",
+    "\n",
+    "In this notebook, the image is loaded and analyzed using a DeepLearning model software [StarDist](https://github.com/stardist/stardist). The StarDist analysis produces a segmentation, which is then converted and saved to the OMERO.server.\n",
```
Comment (pwalczysko) [inline notebooks/idr0062_prediction_save.ipynb]:
```suggestion
    "In this notebook, the image is loaded and analyzed using a Deep Learning software [StarDist](https://github.com/stardist/stardist). The StarDist analysis produces a segmentation, which is then converted and saved to the OMERO.server.\n",
```

**Comment by pwalczysko** [comment]
All works fine, except the RoIs have by default quite a thick yellow stroke and a pink fill, which is surprising.
Screenshot 2022-06-13 at 14 49 39
Screenshot 2022-06-13 at 14 49 34

**Comment by jburel** [comment]
We can change that. I comment that out and use the default

**Comment by jburel** [comment]
@pwalczysko
 I have:
*  fixed the size/color of the shapes
* added viewing of the labels generated by stardist
* added python script matching notebook

**Review by pwalczysko — APPROVED** [review]

**Comment by pwalczysko** [comment]
Thank you

**Pull request merged**